frontend/src/pages/PipelineDetails.tsx

```diff
@@ -111,8 +113,9 @@ class PipelineDetails extends Page<{}, PipelineDetailsState> {
   }
 
   public getInitialToolbarState(): ToolbarProps {
+    const fromRunId = new URLParser(this.props).get(QUERY_PARAMS.fromRunId);
```

Comment:
why not just do an `if`, and then have two `return`s ? Seems like it'd be more straightforward

Reply:
Done.